src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/settings/ManagedIndexSettings.kt

```diff
@@ -24,7 +24,7 @@ import java.util.function.Function
 class ManagedIndexSettings {
     companion object {
         const val DEFAULT_ISM_ENABLED = true
-        const val DEFAULT_JOB_INTERVAL = 5
+        const val DEFAULT_JOB_INTERVAL = 1
```

Comment:
Let's keep default to 5 for now, there are still some other perf improvements we'll want to do before changing default to 1.